src/Layout.tsx

```diff
@@ -6,6 +6,8 @@ import Toast from 'src/components/common/Toast'
 import { useUI } from 'src/sdk/ui'
 import type { PropsWithChildren } from 'react'
 
+import PostalCodeBar from './components/common/PostalCode/PostalCodeBar'
```

Comment:
Do you mind moving this `import` after the `NavBar` one and using the same similar path (`src/components/common/…`) just to keep the same pattern?

![CleanShot 2022-04-05 at 10 37 44](https://user-images.githubusercontent.com/381395/161766760-8aae7f3f-4c90-47a4-a2dd-01d6c5ed293a.png)